### ISSUE TITLE
Issue using 1.7 ores in cofh ore gen. 

### ISSUE THREAD

**Issue by delet3r** [TechReborn/TechReborn#632]
I realize this is not something that a dev needs to deal with, but I doubt i can solve this anywhere but here as this only happens with techreborn ores.  I am using the latest 1.7 tech reborn jar, and reborncore jar.

Basically I am generating 'fractal' ore clusters using cofh core.  Here is the code:

> "tincluster": {
>         "template": "fractal",
>         "block": [{
>             "name": "techreborn:techreborn.ore", "metadata": 4
>             "weight": 20
>         }, {
>             "name": "cobblestone",
>             "weight": 20
>         }, {
>             "name": "ThermalFoundation:Ore", "metadata": 1,
>             "weight": 40
>                 }, {
>             "name": "redstone_ore",
>             "weight": 10
>         }],
>         "material": "minecraft:stone",
>         "clusterSize": 100,
>         "numClusters": 1,
>         "chunkChance": 10,
>         "minHeight": 5,
>         "maxHeight": 12,
>         "veinHeight": 10,
>         "veinDiameter": 10,
>         "verticalDensity": 15,
>         "horizontalDensity": 15,
>         "retrogen": "true",
>         "chunkChance": 4,
>         "biomeRestriction": "none",
>         "biomes": [],
>         "dimensionRestriction": "blacklist",
>         "dimensions": [-1,
>             1
>         ]
>     },   `

I have about 20 different types of these, different ores, different y levels, different sizes, etc.  The entire thing works perfectly until I try to put in a techreborn ore of any type.  For a thermal expansion the ThermalFoundation:Ore:1 is tin.  When using minetweaker it says the name of bauxite ore is techreborn:techreborn.ore:4.  I put it in using the same syntax, "name": "techreborn:techreborn.ore", "metadata": 4 and it does not work.  I put it as the first entry in my .json and it then makes no ores generate at all.  If i remove the techreborn ore name and put another one back in, everything works. 

Is there something different about techreborn ores that would make them not work with cofh core?  

Again, I realize this is not really a 'bug' or an issue anyone here really needs to deal with, but its very hard to find people who have even used cofh core's ore gen much, and beyond that, this issue only appears with the techreborn ores.  So I thought posting here was my only hope.  If I cant get it to work I need to drop techreborn from the pack, the regular ore generation that your mod generates would never work alongside the ore gen ive made here.

**Comment by modmuss50** [comment]
Thats got nothing to do with me, sorry. From what I have heard it works just fine.

**Comment by delet3r** [comment]
Completely understand.  I thought possibly the ores were different than normal, like in later GT versions the ores were tile entities, and would not work with cofh ore gen.  I assume if that was the case you would have let me know, so I must be doing something wrong.  Thanks for your time.

**Comment by modmuss50** [comment]
They are not tiles, they are just normal meta data blocks. I know it works as i think its being used in FTB inventions

**Comment by delet3r** [comment]
Ok that at least proves its something im doing. :)  Thanks!

**Comment by Prospector** [comment]
Take a peak at the FTB inventions configs, that might help

**Comment by delet3r** [comment]
A guy on reddit told me about another pack using tech reborn, Material Energy Natural Capital.  He uses tech reborn ores just fine, no issues.  

I found out I was missing a comma after the line with the TR ore in it.  0_o   Sorry for wasting anyone's time, I was SURE that I checked for syntax errors, but obviously did not.  It works fine now.  

Really hope that especially with the upcoming unity textures that your mod takes off and gets more popular.  With the popularity of Infinity expert, i think the community is ready for more slow paced game play.
